editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1866,7 +1866,32 @@ export const MetadataUtils = {
       return this.findNearestAncestorFlexDirectionChange(elementMap, parentPath)
     }
   },
-  isFocusableComponentFromMetadata(element: ElementInstanceMetadata | null): boolean {
+  isAutofocusable(
```

Comment:
Minor, but this should be `isAutoFocused`, because either it has automatically been focused or it hasn't. Technically all parsed components are "auto-focusable", they just need to be wrapped in a `Scene` to trigger that.

Reply:
What about `isAutofocusableCandidate`? As this is the logic that does the check, as opposed to the one that inspects `autoFocusedPaths`?

Reply:
Hmmm yeah ok that's actually a fair point, as otherwise it would be checking against the list of autofocused paths. In that case I'd say just leave the name as it is then and ignore my suggestion 👍